Add page to list all KEPs (#324)
kubernetes/contributor-site: content/en/resources/keps/_index.md

---
linktitle: Enhancements
title: Kubernetes Enhancement Proposals (KEPs)
description: |

>   List of enhancements that are proposed

Reviewer: ```suggestion
   List of Kubernetes enhancements.
```
“proposed” doesn't capture that some of these are fully implemented.

Author: In essence, all enhancements are proposals in various states of implementation, but I agree that the language can create some confusion.

type: docs
menu:
  main:
    weight: 40
---

These data come from the [k/enhancements][k-enhancements]
repository on GitHub.

> Note: If you notice an empty cell in the table, the KEP may not have updated information.

> Please raise an issue in the [k/enhancements][k-enhancements] repository.

Reviewer: If possible, avoid using a block quote to mark a note. Try this instead:

```suggestion

Reviewer: ```suggestion
{{% alert title="Note" color="info" %}}
If you notice an empty cell in the table, the KEP may not have updated information.
Please raise an issue in the [k/enhancements][k-enhancements] repository.
{{% /alert %}}
```

Author: Done.

Author: This is a nifty TIL for me. Thanks for the tip!

---
linktitle: Enhancements
title: Kubernetes Enhancement Proposals (KEPs)
description: |
  List of Kubernetes enhancements
type: docs
menu:
  main:
    weight: 40
---

These data come from the [kubernetes/enhancements]
repository on GitHub.

> 

Reviewer 2: ```suggestion

To see the original Enhancement issue, click on the KEP number.  To see a summary of what the KEP does, click on the Title.

```

... this behavior is useful, but unintuitive, so adding text explaining it helps.

{{% alert title="Note" color="info" %}}
If you notice an empty cell in the table, the KEP may not have updated information.

> Please raise an issue in the kubernetes/enhancemenets repository.

Reviewer 2: ```suggestion
Please raise [an issue in the kubernetes/enhancements](https://github.com/kubernetes/enhancements/issues) repository.
```
